src/test/java/uk/gov/cshr/repository/InviteRepositoryTest.java

```diff
@@ -83,6 +84,20 @@ public void existsByCodeAndStatusReturnsCorrectResult() {
         assertThat(existsByCodeAndStatusForExpiredInvite, equalTo(false));
     }
 
+    @Test
+    public void findByForEmailAndStatusShouldReturnCorrectInvite() {
+        Invite invite = createInvite();
+
+        inviteRepository.save(invite);
+
+        Optional<Invite> actualInvite = inviteRepository.findByForEmailAndStatus("test@example.org", InviteStatus.PENDING);
+
+        assertThat(actualInvite.get().getCode(), equalTo("123abc"));
+        assertThat(actualInvite.get().getForEmail(), equalTo("test@example.org"));
+        assertThat(actualInvite.get().getStatus(), equalTo(InviteStatus.PENDING));
+
+    }
+
     private Invite createInvite() {
         return createInvite("123abc", "test@example.org");
```

Comment:
Can these two values be assign to constants in the unit test and then use the constants in the create and assert so it is easy to read the test else it is confusing to read asserts in the previous unit test method above.